call-for-speakers-template.md

```diff
@@ -1,41 +1,46 @@
 # Header
 
-To: cool person
-Cc: the other CTFEDs orgs
+**To:** cool person<br>
+**Cc:** the other CTFEDs orgs
 
 # Subject
 
-Talk at CTFEDs?
+Would you give a talk at the CTFEDs meetup?
 
 # Body
 
 Hi! :)
 
 `intro saying hi and stuff`
 
-Short version: **would you give a talk at CTFEDs later this year? :)**
+Short version: would you give a talk at the CTFEDs meetup? `Elevator pitch on why them or what topic or both` :)
+
+Examples / ideas:
+
+- We think you'd be great to give on a talk on React since you're learning it right now.
+- Since you're newish to Front-end, you're in the perfect position to help other people in the same situation. And to remind the old ones what they've forgotten!
+- We think your persective (as a mostly back-end developer) would give helpful insights to front-enders working with back-enders.
 
 Long version below.
 
 ---
 
-We are looking for speakers for our CTFEDs (Cape Town Front-end Developers) [meetup group](https://www.meetup.com/ctfeds/) for this year. We meet after work on the first Wednesday of the month (although when we do workshops we do them on a Saturday morning) and we’ve got open slots from June onwards. Would you be interested in speaking? :)
+We would love you to come and speak at our CTFEDs (Cape Town Front-end Developers) [meetup group](https://www.meetup.com/ctfeds/). We feel that speaking can be a bit initimidating, so we offer as much help as we can. We can help with the topic, the talk (or workshop), help you practice, and more. We’ve got a [Call For Speakers page](http://ctfeds.org/call-for-speakers/) that has a bunch more words about this.
```

Comment:
Maybe we could reword this a little. I don't think we should pre-empt them by saying it is intimidating, but rather that it might be. Subtle, but I think it can make a difference.

"Whether it's your first time talking or your tenth, we understand it might feel a bit intimidating, so we offer as much support as you need."

Reply:
Great! Wanna push that change? :)